lib/informant.rb

```diff
@@ -48,7 +50,7 @@ class Standard < ActionView::Helpers::FormBuilder
     @@custom_options = @@custom_field_options + @@custom_label_options
 
     # Run already-defined helpers through our "shell".
-    helpers = field_helpers +
+    helpers = field_helpers.map(&:to_s) +
```

Comment:
This should be enough to make it compatible with Rails 4. It's good that we have this discussion because this seems like a more suitable "fix".